department-of-veterans-affairs/caseflow
# APPEALS-43678 and APPEALS-36344

Resolves [APPEALS-43678](https://jira.devops.va.gov/browse/APPEALS-43678) and [APPEALS-36344](https://jira.devops.va.gov/browse/APPEALS-36344)

# Description
[APPEALS-43678](https://jira.devops.va.gov/browse/APPEALS-43678): fixed bug where supervisor could not resolve remove package task

[APPEALS-36344](https://jira.devops.va.gov/browse/APPEALS-36344): Fixed bug where superuser could not access the Reassign package task workflow. 

## Acceptance Criteria

## Testing Plan
1. Sign in as MAIL_TEAM_SUPERVISOR_MAIL_INTAKE_USER and navigate to Queue
2. Click Switch Views and select Correspondence Cases
3. Click the **Action Required** tab and select a veteran that has an **Action Type** of **Remove**
4. Click **View package**.
5. There should be a blue banner, and a blue button allowing the user to resolve the action
6. **Before navigating away from the page, copy the URL for the remove package. This will be needed to test the super user workflow**
7. Navigate back to Correspondence Cases, navigate to **action required** and select a veteran with an **Action Type** of **Reassign**.
8. Click **View package**.
9. There should be a blue banner, and a blue button allowing the user to resolve the action

10. Sign in as AMBRISVACO
11. Enter the URL for the Remove Package from step 6 (Superusers have no access to Remove packages on their Correspondence Cases queue).
12. A blue banner should appear on the page and there is **no button** for the user to interact with. The rest of the contents of the page should be locked. 
13. Navigate back to Correspondence Cases, navigate to **action required** and select a veteran with an **Action Type** of **Reassign**.
14. Click **View package**.
15. There should be a blue banner, and a blue button allowing the user to resolve the action

# Frontend
## User Facing Changes

**Supervisor Reassign**
![Supervisor Reassign](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/2f84dd1b-c71e-4b17-9c68-df4d8a0b05e9)

**Supervisor Remove**
![Supervisor Remove](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/6c5ffe0c-84a8-414e-9e8b-6ab00e07c4e1)

**Superuser Reassign**
![Superuser Reassign](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/c2e5a67e-21de-4917-b1fa-322a7b23534c)

**Superuser Remove**
![Superuser Remove](https://github.com/department-of-veterans-affairs/caseflow/assets/99915461/394a915a-bf1d-419e-b414-371e0fa4a297)
